### ISSUE TITLE
Set-Cookie header with ';SameSite;' crashes in analyzer/headers.py

### ISSUE THREAD

**Issue by floatingatoll** [mozilla/http-observatory#369]
```
  File "/usr/local/lib/python3.6/site-packages/httpobs/scanner/analyzer/headers.py", line 336, in cookies
    if cookie._rest[key] is True or cookie._rest[key].strip().lower() == 'strict':
```

Introduced by mozilla/http-observatory#313 and triggered by:

`set-cookie: PHPSESSID=14kqgnsqfm5g3cgcgb592urig0; path=/; HttpOnly;SameSite;Secure`

Reported by @vinzv in mozilla/http-observatory-website#176.

**Comment by april** [comment]
SameSite alone should be fine (it defaults to Strict), this should only generate an error if you have something like `SameSite=foo`.

**Comment by floatingatoll** [comment]
Hmm. In which case my patch won’t help, sorry :(

**Comment by april** [comment]
Fixed in 61e49e8.  :)
